src/main/kotlin/no/skatteetaten/aurora/boober/utils/PropertiesUtil.kt

```diff
@@ -0,0 +1,47 @@
+package no.skatteetaten.aurora.boober.utils
+
+import java.io.ByteArrayInputStream
+import java.io.ByteArrayOutputStream
+import java.io.IOException
+import java.util.*
+
+
+// function that filters out everything except the lines with the provided keys from a .properties ByteArray
+fun filterProperties(properties: ByteArray, keys: List<String>) : ByteArray {
```

Comment:
Hva med
`fun filterProperties(propertyBytes: ByteArray, keys: List<String>): ByteArray =
        loadFromByteArray(propertyBytes)
                .filter(keys)
                .toByteArray()
fun loadFromByteArray(properties: ByteArray): Properties {
    return Properties().also {
        try {
            it.load(ByteArrayInputStream(properties))
        } catch (ioe: IOException) {
            throw PropertiesException("Encountered a problem while reading properties.", ioe)
        } catch (iae: IllegalArgumentException) {
            throw PropertiesException("Encountered a problem while reading properties. The input stream contains a " +
                    "malformed Unicode escape sequence.", iae)
        }
    }
}
fun Properties.filter(keys: List<String>): Properties {
    val propertyNames = this.stringPropertyNames()
    val newProps = Properties()
    keys.filter { propertyNames.contains(it) }
            .map { it to this.getProperty(it) }
            .forEach { (key, value) -> newProps.put(key, value) }
    return newProps
}
fun Properties.toByteArray(): ByteArray = ByteArrayOutputStream()
        .let { baos ->
            this.store(baos, "Properties filtered.")
            baos.toByteArray()
        }`